lib/feeder/sync_episodes.ex

```diff
@@ -1,28 +1,34 @@
 defmodule Feeder.SyncEpisodes do
-
   def sync(), do: sync(nil)
+
   def sync(dtim) do
     case get_episodes(dtim) do
-      {:error, err} -> {:error, err}
+      {:error, err} ->
+        {:error, err}
+
       {_ok_or_partial, total, docs} ->
         {created, updated} = with_podcast(docs) |> update_episodes()
         {:ok, created, updated, total - created - updated}
     end
   end
 
-  defp get_episodes(nil), do: Feeder.Api.episodes()
+  defp get_episodes(nil) do
+    Ecto.Adapters.SQL.query!(Castle.Repo, "UPDATE episodes SET updated_at = NULL")
```

Comment:
Note: the `mix feeder.sync --force` flag wasn't actually working before.  It would do a query `?since=1970-01-01`, update those in the db, then `SELECT max(updated_at) from episodes`.  Which would return something very very recent, and jump to `?since=2020-04-09` ish.  Skipping everything between.

Clearing the updated_at forces it to refresh _everything_ in the episodes table.